forge/db/migrations/20230816-01-add-ApplicationId-to-Devices.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-unused-vars */
```

Comment:
This file needs to be renamed to ensure it is the most recent migration in the list.

At the time of commenting, the most recent migration is `20230818-01-add-access-token-name.js`

Reply:
@knolleary resolved in [9f9b53f](https://github.com/flowforge/flowforge/pull/2621/commits/9f9b53f46ef545d82102e3ff9974ec6725ca456f)

Reply:
Not resolved - we now how two migration files :) Need to delete the old one.

Reply:
oh, odd, i did a rename of file and (wrongly) expected that to remove the old one. 

Will fix now.

Reply:
Double odd

![image](https://github.com/flowforge/flowforge/assets/44235289/0bdc2d21-fd58-47c6-b587-9e896cd1756c)

Reply:
Doh! 👐 

![image](https://github.com/flowforge/flowforge/assets/44235289/5d9ae878-d432-4a0a-8c68-2886282d0034)

Reply:
Looks good - resolving this thread.